### PR TITLE
Modified lint command to accept multiple arguments

### PR DESCRIPTION
**Pull request by sapzape**
This change fixes the lint command apply to client/ directory as well.

ISSUE=#533

### CODE REVIEW

**Comment by sapzape** [comment]
@romandev 
I also seem to be unable to handle errors right away...
I think it would be better to divide the patch as you say.

**Comment by sapzape** [comment]
In this patch, i will add a lint command for the client side.

**Comment by romandev** [comment]
@sapzape any update?

**Comment by sapzape** [comment]
@romandev PTAL

File: gulpfile.ts [hunk 1]
```diff
@@ -42,10 +39,28 @@ gulp.task('lint', () => {
...
+  .pipe(tslint({
+    fix: true,
+    formatter: 'codeFrame'
+  }))
+  .pipe(tslint.report({
+    summarizeFailureOutput: true
+  }));
+});
+
+gulp.task('lint_client', () => {
```
Comment (romandev) [inline gulpfile.ts]: Let's leave a FIXME comment to remember to remove this command later.
(If possible, please file a issue as well.)

File: gulpfile.ts [hunk 2]
```diff
@@ -42,10 +39,28 @@ gulp.task('lint', () => {
...
+  gulp.src('./client/**/*.ts')
+  .pipe(tslint({
+    formatter: 'codeFrame'
+  }))
+  .pipe(tslint.report({
+    summarizeFailureOutput: true
+  }));
+});
+
+gulp.task('lint_client:fix', () => {
```
Comment (romandev) [inline gulpfile.ts]: ditto.

**Comment by sapzape** [comment]
@romandev PTAL

**Pull request merged**